djangopg/fields.py

```diff
@@ -73,20 +72,20 @@ def db_type(self, connection):
         return 'citext'
 
     def to_python(self, value):
-        if isinstance(value, unicode) or value is None:
+        if isinstance(value, six.text_type) or value is None:
             return value
         if isinstance(value, str):
```

Comment:
This line may refer to binary type, on Python 3:
```
>>> value = str('123')
>>> if isinstance(value, str):
...     value.decode('UTF-8')
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
AttributeError: 'str' object has no attribute 'decode'
```
Please, make sure the coverage of this library is 100%, this library is critical.